vulnerable_people_form/form_pages/shared/session.py

```diff
@@ -352,7 +352,11 @@ def get_location_tier():
 
 
 def set_is_postcode_in_england(is_postcode_in_england):
-    session[SESSION_KEY_IS_POSTCODE_IN_ENGLAND] = is_postcode_in_england
+    postcode = session["postcode"]
```

Comment:
i would move this logic in the location_tier.py just like the above change made for `update_is_postcode_in_england`